config.go

```diff
@@ -172,6 +173,20 @@ func (h hypervisor) defaultMemSz() uint32 {
 	return h.DefaultMemSz
 }
 
+func (h hypervisor) defaultBridges() uint32 {
+	if h.DefaultBridges <= 0 {
+		return defaultBridgesCount
+	}
+
+	// To allow other devices be plugged directly in the pci(e).0
+	// until 10 bridges can be plugged per POD/VM.
+	if h.DefaultBridges > 10 {
```

Comment:
- I don't understand this limit. What if I want `11` bridges for my pod?
- Can you either expand the comment to explain this in more detail, or mention that more details are provided in the config file?
- How about creating a const for the `10`?
- If we return `10` here, I think we should emit a log message explaining what is happening since we're overriding what the user requested in the config file.

Reply:
Actually , I can plug until 14 bridges in Q35 and 8 in PC, It could be an limitation not documented on Qemu or We have an issue in the kernel.
Anyway, I'll change this to 5 and document this limitation